sdk/include/c++/stlport/stl/config/_reactos.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2010
- * J�r�me Gardou
+ * Jérôme Gardou
```

Comment:
does that change fix or break the encoding in this file?

Reply:
Yes that makes it UTF-8.